<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPropertyDialog</class>
 <widget class="QWidget" name="GUIPropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Property Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelType">
     <property name="text">
      <string>Property Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditType"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPropertyID">
     <property name="text">
      <string>Property KeyWord</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditPropertyID">
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QTableWidget" name="tableWidgetValue">
     <property name="cornerButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="rowCount">
      <number>15</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>Value</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
